doc/sphinx/introduction.rst

```diff
@@ -64,5 +64,3 @@ This manual is organized in three main parts, plus an appendix:
   :ref:`indexes <indexes>` are very useful to **quickly browse the
   manual and find what you are looking for.** They are often the main
   entry point to the manual.
-
-The full table of contents is presented below:
```

Comment:
Why do you remove that? (in the generated refman, we indeed get the table of content below)

Reply:
If I open the html I indeed get that:
![image](https://github.com/coq/coq/assets/109028294/f2784032-2db4-43da-9a32-2267aaedc242)

However, if I open the pdf downloaded from the release page (left page first):
![image](https://github.com/coq/coq/assets/109028294/48c27431-6ef2-45db-bd92-3a8ec54a35b2)

Here the ToC comes before chapter 1 / the introduction and so the sentence makes no sense.

Instead of finding out how to reshuffle the order of pages somewhere in the build system (which someone who knows their way around might do instead if you reject this commit) I just removed the dependency. On the html version the heading `Contents` is more than enough, you don't need to write it out in a different section too.

Reply:
Indeed but the PDF manual is probably much less used so I don't know what we want to do @coq/doc-maintainers any opinion?

Reply:
I understand it's less used, which is why I think the simple fix of removing a useless sentence is fine.

Currently we have the html saying:

> Table of contents is below:
> # Contents

Which is redundant.

Reply:
It's not redundant, Contents being the title of the first chapter. HTML being probably much more used than PDF, I'd rather keep the sentence than remove it. There is likely a better solution though.

Reply:
I'm OK with dropping the sentence.  I don't think anyone will be confused by its absence.

Reply:
I don't remember the exact syntax, but there are ways to make paragraphs media-specific, which would also solve the issue.

Reply:
```rest
.. only:: html

   The full table of contents is presented below:
```